chainercv/links/model/sequential_feature_extractor.py

```diff
@@ -9,11 +7,11 @@ class SequentialFeatureExtractor(chainer.Chain):
 
     This class is a base class that can be used for an implementation of
     a feature extractor model.
-    The link takes an argument :obj:`layers` that specifies the computation
-    conducted in :meth:`__call__`.
-    :obj:`layers` is a :class:`collections.OrderedDict` of
-    callable objects called layers, which are going to be called sequentially
-    starting from the top to the bottom.
+    Callabel objects, such as :class:`chainer.Link` and
+    :class:`chainer.Function`, can be registered to this link with
```

Comment:
This is a problem from the original doc, but can you change the phrase `this link` to `this chain`?